client/landing/jetpack-cloud/components/daily-backup-status/index.jsx

```diff
@@ -318,12 +336,39 @@ class DailyBackupStatus extends Component {
 		return this.renderNoBackupOnDate();
 	}
 
+	renderRealtimeDetails( backup ) {
+		const { moment, allowRestore, timezone, gmtOffset, siteSlug } = this.props;
+
+		return (
+			<div className="daily-backup-status__realtime-details">
+				<div className="daily-backup-status__realtime-details-title">Backup details</div>
+				<div className="daily-backup-status__realtime-details-card">
+					<ActivityCard
+						{ ...{
+							moment,
+							activity: backup,
+							allowRestore,
+							timezone,
+							gmtOffset,
+							siteSlug,
+							summarize: true,
+						} }
+					/>
+				</div>
+			</div>
+		);
+	}
+
 	render() {
-		const backup = this.props.backup;
+		const { dailyBackup, hasRealtimeBackups, realtimeBackups } = this.props;
+		const backup = hasRealtimeBackups ? realtimeBackups[ 0 ] : dailyBackup;
 
 		return (
 			<div className="daily-backup-status">
-				<Card className="daily-backup-status__success">{ this.renderBackupStatus( backup ) }</Card>
+				<Card className="daily-backup-status__success">
+					{ this.renderBackupStatus( backup ) }
+					{ hasRealtimeBackups && this.renderRealtimeDetails( backup ) }
```

Comment:
`backup` could be null, so we'll get an error. Since this is part of a good backup block, let's move this line inside of `renderGoodBackup`, so we ensure we have a valid backup.